### PR TITLE
SW-5791 Fix some issues on Modules' Events

### PR DESCRIPTION
**Pull request by constanzauanini**

### CODE REVIEW

File: src/scenes/AcceleratorRouter/Modules/EventsCellRenderer.tsx [hunk 1]
```diff
@@ -23,6 +23,22 @@ export default function EventsCellRenderer(props: RendererProps<TableRowType>):
...
+  if ((column.key === 'meetingUrl' || column.key === 'recordingUrl' || column.key === 'slidesUrl') && !onRowClick) {
+    return (
+      <CellRenderer
+        index={index}
+        column={column}
+        value={
+          <Link fontSize='16px' target='_blank' to={value as string}>
+            {' '}
+            {value as string}{' '}
+          </Link>
```
Comment (joshbuchea) [inline src/scenes/AcceleratorRouter/Modules/EventsCellRenderer.tsx:35]: I assume the rendered spaces are intentional to achieve some desired spacing?

**Review by joshbuchea — APPROVED** [review]
Looks good, just one question

File: src/scenes/AcceleratorRouter/Modules/EventsCellRenderer.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/scenes/AcceleratorRouter/Modules/EventsCellRenderer.tsx:35]: Oh, no, thanks for noticing this.

Comment (joshbuchea) [reply src/scenes/AcceleratorRouter/Modules/EventsCellRenderer.tsx]: 🙌

**Pull request merged**